test/e2e/common.go

```diff
@@ -274,7 +274,7 @@ func EnsureControlPlaneInitialized(ctx context.Context, input clusterctl.ApplyCl
 	}, input.WaitForControlPlaneIntervals...).Should(Succeed(), "API Server was not reachable in time")
 
 	_, hasWindows := cluster.Labels["cni-windows"]
-	if kubeadmControlPlane.Spec.KubeadmConfigSpec.ClusterConfiguration.ControllerManager.ExtraArgs["cloud-provider"] != "azure" {
+	if false {
```

Comment:
why this change? the ipv6 job needs to install CCM

Reply:
I'd like to check what will happen without CCM since the impacted Pods should exist before CCM is deployed.